course/frontend/camel-case-to-lower-case-with-spaces/scripts/main.js

```diff
@@ -1 +1,6 @@
-// TODO
\ No newline at end of file
+function camelCase(text) {
```

Comment:
well done @JaimeRC just a little detail: move this function to camelCase.js file. WARN! this file already exists in upstream, so before doing it, fetch upstream and merge upstream/develop into your branch. then ADD your implementation but DONT remove the other implementations (from other mates).